### PR TITLE
Update molpay.rb

### PR DESCRIPTION
**Pull request by frankiecmk**
- to support pending and unknown status.

### CODE REVIEW

**Comment by andrewpaliga** [comment]
@frankiecmk can you write a test?

**Comment by frankiecmk** [comment]
@andrewpaliga I have added the test case, let me know if anything wrong.

File: test/unit/integrations/molpay/molpay_return_test.rb [hunk 1]
```diff
@@ -13,13 +13,18 @@ def setup
   def test_success?
     assert @molpay.success?
   end
-
+
```
Comment (andrewpaliga) [inline test/unit/integrations/molpay/molpay_return_test.rb]: white space added here, can you remove?

**Comment by andrewpaliga** [comment]
You'll also want to add a test here: https://github.com/frankiecmk/offsite_payments/blob/patch-7/test/unit/integrations/molpay/molpay_notification_test.rb

**Comment by frankiecmk** [comment]
@andrewpaliga I have remove the white spacing and added test case for notification.

**Comment by andrewpaliga** [comment]
@aprofeit @ThereExistsX review?

File: lib/offsite_payments/integrations/molpay.rb [hunk 2]
```diff
@@ -176,6 +183,10 @@ def initialize(query_string, options = {})
         def success?
           @notification.acknowledge
         end
+        
+        def pending?
+          params['status'] == "22"
```
Comment (aprofeit) [inline lib/offsite_payments/integrations/molpay.rb]: Should this use the new `status` method? Then we'll only be parsing `params['status']` in one place.
```
def pending?
  status == 'Pending'
end
```

**Comment by frankiecmk** [comment]
@aprofeit I have amend it.

**Comment by aprofeit** [comment]
Merged, thanks!

**Pull request closed without merging**